src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,37 @@
+package wolox.training.models;
+
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Data
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
```

Comment:
Nice use of Lombok! 😎  Remember to add this line `@Setter(AccessLevel.NONE)` to prevent the id from being modified after generation 😄